directory/cli/src/directory.py

```diff
@@ -60,7 +61,20 @@ def directory():
     # maybe move calling so only done if needed.
     utils.obtain_kerberos_ticket()
 
-command_modules = standard_command_modules + [user, group, import_export, host, hostgroup]
+if utils.advanced_mode_enabled():
+    command_modules = standard_command_modules + [
+                user,
+                group,
+                import_export,
+                host,
+                hostgroup,
+            ]
+else:
+    command_modules = standard_command_modules + [
+                user,
+                group,
+                import_export,
+            ]
```

Comment:
I imagine this will be changing in future PRs, and so this comment might be outdated, but if this is still relevant: this conditional could probably be made less duplicated - as it stands right now this could be simplified something like:
```python
command_modules = standard_command_modules + [
    user,
    group,
    import_export,
]
if utils.advanced_mode_enabled():
    command_modules += [host, hostgroup]
```